nubia/internal/typing/argparse.py

```diff
@@ -99,7 +99,7 @@ def register_command(argparse_parser, inspection):
     # auto wrap the function with @command in case its not wrapped into one
     subparsers = _resolve_subparsers(argparse_parser)
 
-    subparser = subparsers.add_parser(_command.name, help=_command.help)
+    subparser = subparsers.add_parser(_command.name, aliases=_command.aliases, help=_command.help)
```

Comment:
@marimar96, could you please fix the formatting here?
```python
subparser = subparsers.add_parser(
    _command.name, aliases=_command.aliases, help=_command.help
)
```
Sorry about that, we need to set up a pre-commit hook that will run the linters.

Reply:
Ok, will do it!

Reply:
I was seeing what the problem was by doing this:
Downloadin again the repo because of the changes and ran the tests (without including my changes previosly commited to try to find out the problem) that failed because of this:

FAIL: test_function_to_str (helpers_test.HelpersTest)

Traceback (most recent call last):
  File "/home/travis/build/facebookincubator/python-nubia/tests/helpers_test.py", line 26, in test_function_to_str
    test("tests.helpers_test.foo", True, False)
  File "/home/travis/build/facebookincubator/python-nubia/tests/helpers_test.py", line 23, in test
    self.assertEqual(function_to_str(foo, with_module, with_args), expected)
AssertionError: 'helpers_test.foo' != 'tests.helpers_test.foo'
- helpers_test.foo
+ tests.helpers_test.foo
? ++++++

As I thought it was nothing to do with my changes then saw this :  *fix tests and bump version 0.2b1 #47*. So made the changes and ran out again the tests and finishes successfully. What should I do then, cuz only added spaces on the code that previously what all good and checked.

Reply:
@marimar96, I believe your changes are just fine to be merged while #47 will fix the tests.